internal/aliasfix/mappings.go

```diff
@@ -750,6 +750,14 @@ var GenprotoPkgMigration map[string]Pkg = map[string]Pkg{
 		ImportPath: "cloud.google.com/go/firestore/apiv1/firestorepb",
 		Status:     StatusInProgress,
 	},
+	"google.golang.org/genproto/googleapis/iam/v1": {
+		ImportPath: "cloud.google.com/go/iam/apiv1/iampb",
+		Status:     StatusInProgress,
```

Comment:
Should this be `StatusMigrated`, since `iampb` is included in this PR?

Reply:
nope, we still need to alias over to the new stubs since the current stubs are under use.